packages/frontend/src/app/home/home.component.html

```diff
@@ -17,31 +17,52 @@
 <!--PRESENTATION TEXT-->
 <div id="presentation-text">
     <h2>Welcome to flaky.dev !</h2>
-    <h3>
-        <p>This is a platform that allows you to track every test in your repository.</p>
-        <p>If any of them is flaky, it will be found !</p>
-    </h3>
 </div>
 
-<!--INTRODUCTION TEXT-->
-<div id="introduction-text">
-    <p>What are you going to do now?</p>
-</div>
 
-<!--SEARCH BAR-->
-<div id="search-bar">
-    <app-search (searchOptionSelected)="onSearchOptionSelected($event)"></app-search>
+<!--DESCRIPTIONS-->
+<div class="description">
+
+    <section class="description-section">
+        <h3 class="description-header">All your tests in one place</h3>
+        <p class="description-text">
+            Get an organization-wide insights on the test health of your repositories.
```

Comment:
"Get organization-wide insights into the test health of your repositories" maybe?